Sources/protoc-gen-swift/MessageGenerator.swift

```diff
@@ -67,6 +68,9 @@ class MessageGenerator {
       swiftRelativeName = sanitizeMessageTypeName(file.swiftPrefix + descriptor.name)
       swiftFullName = swiftRelativeName
     }
+    self.isAnyMessage = (isProto3 &&
+                         descriptor.name == "Any" &&
```

Comment:
The name and file check came from the C++ code, but adding package since I have no problem being overly paranoid (the leading `.` isn't there though).